Bring disparity b/w the form validation lesson in Intermediate HTML/CSS and the form lesson in JavaScript course (#23387)
TheOdinProject/curriculum: html_css/intermediate_html/form-validations.md

Validations are a vital ingredient in well-designed forms. They help protect our backend systems from receiving incorrect data, and they help make the experience of interacting with our form as dead-stupid-simple as possible for our users.

> This lesson will explore some of the built-in validations you can use with HTML forms.

Reviewer: This should include mention that the user will also learn about styling for validations.

<div class="lesson-content__panel" markdown="1">

1. Read and follow along to [MDN's Client-Side Form Validation Guide](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation)

> 2. SitePoint's [Complete Guide to HTML Forms and Constraint Validation Guide](https://www.sitepoint.com/html-forms-constraint-validation-complete-guide/)

Reviewer: I think this could either stay here as an assignment item, if only for the CSS Validation Styling section in the article, or it could make a good additional resource. It does cover some of the same stuff as the MDN article albeit slightly differently, and again the CSS validation styling I think could be useful for users to read about.

Author: I've added it back to the assignment section

>     * Note: Halfway through, the doc covers the Constraint Validation API: a way to validate forms with javascript. We'll cover that in a future lesson, you can skip that part for now.

Reviewer: This doesn't need the "Note" text:

```suggestion
    * Skip the section on "Validating forms using JavaScript". This will be covered in a future lesson.
```
